cds_ils/importer/serializers/schema.py

```diff
@@ -119,12 +119,23 @@ def __init__(self, record_offset, **kwargs):
     @post_dump
     def records_statuses(self, data, **kwargs):
         """Return correct record statuses."""
+        import pprint
+        printer = pprint.pprint
         children_entries_query = ImportRecordLog.query \
             .filter_by(import_id=data.get('id'))
+
+        first_entry = children_entries_query.first()
+
+        initial_entry_id = first_entry.id if bool(first_entry) else 0
+
         entries = children_entries_query \
-            .filter(ImportRecordLog.id >= self.records_offset) \
+            .filter(ImportRecordLog.id >= initial_entry_id + self.records_offset) \
```

Comment:
What do you mean? I don't think that there are any funny interactions between threads / requests or maybe i am missing something :thinking:

Reply:
I mean the ids of ImportRecordLog objects are increased in two different imports, so the currently checked will have some gaps in the ids. Did you check the behaviour?

Reply:
I can't quite understand that. The id of a the first item in task will always remain the same because 2 imports can not alter each other because they will need to have 2 different taskIds